changelog_unreleased/typescript/11721.md

```diff
@@ -49,7 +49,7 @@ class Foo {
 
 ```
 
-##### [Import Assertions](https://devblogs.microsoft.com/typescript/announcing-typescript-4-5-rc/#import-assertions)
+##### [Import Assertions](https://devblogs.microsoft.com/typescript/announcing-typescript-4-5/#import-assertions)
 
 <!-- prettier-ignore -->
 ```ts
```

Comment:
I can’t comment on the relevant line below, but I would word it as:

> Prettier will now parse files with `.mts` and `.cts` extensions as TypeScript.

(parse? see? handle? treat? not sure what’s best)

Reply:
format?

Reply:
That works!